packages/react-email/package.json

```
@@ -34,6 +34,7 @@
     "chokidar": "3.5.3",
     "commander": "9.4.1",
     "cpy": "8.1.2",
+    "detect-package-manager": "^2.0.1",
```

Can we pin the version here?

```suggestion
    "detect-package-manager": "2.0.1",
```